docs/spec/data-plane.md

```diff
@@ -91,6 +91,25 @@ delivered.
 CloudEvents received by Sink MAY have
 [Distributed Tracing Extension Attribute](https://github.com/cloudevents/spec/blob/v1.0/extensions/distributed-tracing.md).
 
+### HTTP Attributes
+
+An event sender, including Source and Broker and Channel, MUST include
+all the non-default HTTP attributes with header key `K-Eventing-Http-Attr` in
+every event delivery to specify
+what HTTP features it can support. Without a certain attribute, the receiver
+MUST assume the corresponding HTTP feature is not supported. A list of HTTP
```

Comment:
You can not add MUST here, it is a breaking change to all past versions of Knative.

Reply:
Done